src/mdlComponentHandler.js

```diff
@@ -351,9 +358,15 @@ componentHandler = (function() {
       upgrades.splice(componentPlace, 1);
       component.element_.setAttribute('data-upgraded', upgrades.join(','));
 
-      var ev = document.createEvent('Events');
-      ev.initEvent('mdl-componentdowngraded', true, true);
-      component.element_.dispatchEvent(ev);
```

Comment:
You're right, I did remove it on the downgrade event! Thanks for the keen eye, @tleunen!